doc/source/train/examples/lightning/lightning_mnist_example.ipynb

```diff
@@ -686,7 +977,157 @@
    ]
   },
   {
-   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "You can create a set of Ray Actors for distributed inference using multiple GPUs. Here we demonstrate how to perform distributed testing with 4 GPUs."
```

Comment:
Add a new header before this section to make it clear that it is a different task.